### PR TITLE
Stilyng

### PR DESCRIPTION
**Pull request by Danfall369**
Hello there Code Reviewer

I hope this message finds you well. I have completed the following requirements for the project:

* Added all missing JSX elements to the project without adding any extra functionality
* Matched the final design to Zeplin's design, including all elements and styling
* Used CSS rules detailed in Zeplin to style the website.

Please review my changes and let me know if there are any further adjustments or improvements that need to be made. I appreciate the opportunity to contribute to this project and I look forward to hearing your feedback.

### CODE REVIEW

File: src/components/Books.jsx [hunk 1]
```diff
@@ -48,11 +48,30 @@ const Books = () => {
...
+                <button type="button" className="action-buttons" onClick={handleRemove}>Remove</button>
+                <button type="button" className="action-buttons">Edit</button>
+              </div>
+            </div>
+            <br />
+            <div className="Status-Info" />
+            <hr className="circle" />
+            <div className="status">
+              <h2 className="percent">0%</h2>
+              <p className="completed">Completed</p>
```
Comment (Munsa1) [inline src/components/Books.jsx:68]: - [ ] Kindly fix your design to match the given design. The progress bar that shows the completed percentage is not properly implemented

Your design
image

Given design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Danfall369, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.jsx (hunk 1 above)
Comment (Danfall369) [reply src/components/Books.jsx]: @Munsa1 Done

File: src/styles/Books.css [hunk 2]
```diff
@@ -1,7 +1,147 @@
...
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  border: none;
+  background: none;
+  cursor: pointer;
+}
+
+.status {
+  display: flex;
```
[the remark this answers is not in the document]
Comment (Danfall369) [reply src/styles/Books.css:66]: @Munsa1  Done

**Review by franclobo — APPROVED** [review]
Hi @Danfall369, 

# STATUS: APPROVE 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Git flow ✔️ 
- Good styles ✔️ 

## Optional suggestions

- **[OPTIONAL]** Please 🙏🏽 use the font family Monserrat and RobotoSlab to match with Zeplin design.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**